### ISSUE TITLE
Not able to handle empty response body case

### ISSUE THREAD

**Issue by braj1999** [loopbackio/strong-soap#671]
# Description/Steps to reproduce
- We are working on Workday Human Resource service, and trying out `Inactivate_Organization` operation - https://community.workday.com/sites/default/files/file-hosting/productionapi/Human_Resources/v41.1/Inactivate_Organization.html
- The operation gives response with empty body. please see below
```
<?xml version='1.0' encoding='UTF-8'?><env:Envelope xmlns:env="http://schemas.xmlsoap.org/soap/envelope/">

<env:Body></env:Body>

</env:Envelope>
```
- this is currently failing in `strong-soap` at [link](https://github.com/loopbackio/strong-soap/blob/master/src/client.js#L332) and we are are getting error saying that `Cannot parse response`.

# Link to reproduction sandbox
I am not sure if I can share the actual request as it has some IDs in my org.

# Expected result
- Strong-soap should allow response with no body.

# Additional information

**Comment by dhmlau** [comment]
@braj1999, your PR has landed and `strong-soap@4.0.0` was released. Thanks again for your contribution!

**Comment by braj1999** [comment]
Many thanks @dhmlau
